Maui/MLTrainer/AppShell.xaml.cs

```diff
@@ -1,9 +1,55 @@
-﻿namespace MLTrainer;
+﻿using Microsoft.UI.Xaml;
```

Comment:
Yeah, this is a weird error:
```
AppShell.xaml.cs(1,17): error CS0234: The type or namespace name 'UI' does not exist in the namespace 'Microsoft' (are you missing an assembly reference?)
```
Maybe this happens on Mac?

Reply:
oh yeah makes sense, that api is windows only

Reply:
@jonathanpeppers  fixed!